Octokit.Tests.Integration/Reactive/ObservableRespositoryCommitsClientTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Octokit.Reactive;
+using System.Reactive.Linq;
+using Xunit;
+
+namespace Octokit.Tests.Integration.Reactive
+{
+    class ObservableRespositoryCommitsClientTests
```

Comment:
Make this class public, I think the test discoverer will miss these currently...